VideoLocker/res/layout/fragment_my_all_videos.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Using `<layout>` as the top level tag enables you to access this layout via the [data binding](https://developer.android.com/topic/libraries/data-binding/index.html) API